src/gui/uithememanager.cpp

```diff
@@ -54,6 +55,7 @@ void UIThemeManager::initInstance()
 }
 
 UIThemeManager::UIThemeManager()
+    : m_applicationPalette(new ApplicationPalette)
```

Comment:
I think you can use `m_applicationPalette(new ApplicationPalette(this))` and then you can omit the destructor in this class.

Reply:
QWidget requires QWidget as parent, UIThemeManager is QObject

Reply:
oops, never mind.